recipes/pandas/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version="0.18.1" %}
+
+package:
+    name: pandas
+    version: {{ version }}
+
+source:
+    fn: pandas-{{ version }}.tar.gz
+    url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
+    sha256: 1f3136f69a58d490ba9f9874e67601042a11138270515dedcf6979c003bba79b
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --root=/
```

Comment:
I'm not trying to drive anyone crazy. Am merely trying to preserve the consistency that I was taught when entering this ecosystem. This is sort of thing that we should make a decision on before switching it out part way.

Plus the arguments asked for here were in the last commented diff so they could be copied and pasted. So, I'm not really understanding the frustration or problem here.